utility_analysis_new/dp_engine.py

```diff
@@ -38,8 +37,22 @@ def aggregate(self,
         self._is_public_partitions = public_partitions is not None
         result = super().aggregate(col, params, data_extractors,
                                    public_partitions)
+        error_aggregators = self._create_aggregate_error_compound_combiner(
+            params)
         self._is_public_partitions = None
-        return result
+        # TODO: Implement combine_accumulators (w/o per_key)
+        keyed_by_same_key = self._backend.map(
+            result, _replace_with_same_key, "Rekey partitions by the same key")
```

Comment:
Nit: lambda x: (None, x[1]) 

`None` better empasizes that the key is dummy :), and also (at least to some my experimentation) has smaller serialization size than integers.